### PR TITLE
fix: do not hide pdisk and vdisk popups if mouse on popup content

### PR DESCRIPTION
**Pull request by astandrik**
Closes https://github.com/ydb-platform/ydb-embedded-ui/issues/1360

[Stand](http://ydb-vla-dev02-005.search.yandex.net:8765/monitoring/cluster/storage)

### Test Status: <span style="color: orange;">⚠️ FLAKY</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1435/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 128 | 123 | 0 | 5 | 0 |

### Bundle Size: ✅
Current: 78.97 MB | Main: 78.97 MB
Diff: +0.00 MB (0.00%)

✅ Bundle size unchanged.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

**Comment by artemmufazalov** [comment]
Please, change PR title prefix to `fix:`, so it will appear in Changelog

**Review by artemmufazalov — APPROVED** [review]

**Comment by astandrik** [comment]
@artemmufazalov added debounce cancelation to prevent appearing not needed popups on fast mast movement

**Review by artemmufazalov — APPROVED** [review]

**Comment by astandrik** [comment]
@Raubzeug added delay for table cells as discussed

**Comment by astandrik** [comment]
Deployed stand for testing

File: src/components/VDiskPopup/VDiskPopup.tsx [hunk 1]
```diff
@@ -136,6 +136,13 @@ interface VDiskPopupProps extends PopupProps {
 
 export const VDiskPopup = ({data, ...props}: VDiskPopupProps) => {
     const isFullData = isFullVDiskData(data);
+    const [isPopupOpen, setIsPopupOpen] = React.useState(props.open);
```
Comment (Raubzeug) [inline src/components/VDiskPopup/VDiskPopup.tsx:139]: same here

**Review by Raubzeug — CHANGES_REQUESTED** [review]

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**